cacheops/invalidation.py

```diff
@@ -18,16 +18,28 @@
 __all__ = ('invalidate_obj', 'invalidate_model', 'invalidate_all', 'no_invalidation')
 
 
+def delete_invalid_caches(conj_keys):
+    for conj_key in conj_keys:
+        cache_keys = redis_client.sscan_iter(conj_key, count=1000)
+        for batch in cache_keys:
+            redis_client.srem(conj_keys, batch)
+            redis_client.delete(batch)
+    redis_client.delete(conj_keys)
+
+cleanup_fn = getattr(settings, 'CACHEOPS_CLEANUP_FN', delete_invalid_caches)
```

Comment:
I don't want to add RQ as a requirement in cacheops. This way, the application using cacheops can pass in a function as a setting that replaces the default invalidation function above – in Rover's case, it'll be a function that queues an RQ task.